.gitignore

```diff
@@ -25,3 +25,5 @@ notes*
 
 /public/assets
 /build
+
+.DS_Store
```

Comment:
Missing a newline at the end of the file?  (May need to add that config to your editor).